lib/linux_admin/network_interface.rb

```diff
@@ -112,7 +114,11 @@ def netmask6(scope = :global)
     #
     # @return [String] IPv4 gateway address
     def gateway
-      @network_conf[:gateway]
+      @network_conf[:gateway4]
+    end
+
```

Comment:
I guess it's easier for the commands that use `ip addr`, but `ip route` doesn't take an interface name ... Is there something we can do to make that less fragile?